src/main/java/step1/Operator.java

```diff
@@ -0,0 +1,49 @@
+package step1;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.function.BiFunction;
+
+public enum Operator {
+
+    PLUS("+", Integer::sum),
+    SUBTRACT("-", Operator::subtract),
+    MULTIPLE("*", Operator::multiple),
+    DIVIDE("/", Operator::divide)
+    ;
+
+    private String symbol;
+
+    private BiFunction<Integer, Integer, Integer> operate;
+
+    Operator(final String symbol, final BiFunction<Integer, Integer, Integer> operate) {
+        this.symbol = symbol;
+        this.operate = operate;
+    }
+
+    private static Integer subtract(Integer a, Integer b) {
+        return a - b;
+    }
```

Comment:
wrapper 타입이 꼭 필요한 경우가 아니라면 원시 타입 사용하시는 걸 추천드립니다
```suggestion
    private static int subtract(Integer a, Integer b) {
        return a - b;
    }
```